gateway/src/test/java/org/georchestra/gateway/rabbitmq/SendMessageRabbitmqIT.java

```diff
@@ -0,0 +1,147 @@
+package org.georchestra.gateway.rabbitmq;
+
+import lombok.NonNull;
+import org.geonetwork.testcontainers.postgres.GeorchestraDatabaseContainer;
+import org.georchestra.ds.orgs.OrgsDao;
+import org.georchestra.ds.users.AccountDao;
+import org.georchestra.gateway.accounts.admin.AccountCreated;
+import org.georchestra.gateway.accounts.admin.CreateAccountUserCustomizer;
+import org.georchestra.gateway.accounts.events.rabbitmq.RabbitmqAccountCreatedEventSender;
+import org.georchestra.gateway.app.GeorchestraGatewayApplication;
+import org.georchestra.security.api.UsersApi;
+import org.georchestra.security.model.GeorchestraUser;
+import org.georchestra.testcontainers.console.GeorchestraConsoleContainer;
+import org.georchestra.testcontainers.ldap.GeorchestraLdapContainer;
+import org.georchestra.testcontainers.rabbitmq.RabbitmqContainer;
+import org.georchestra.testcontainers.smtp.SmtpContainer;
+import org.hamcrest.Matchers;
+import org.junit.ClassRule;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.amqp.rabbit.connection.CachingConnectionFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.reactive.AutoConfigureWebTestClient;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.system.CapturedOutput;
+import org.springframework.boot.test.system.OutputCaptureExtension;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.ldap.NameNotFoundException;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.web.reactive.server.WebTestClient;
+import org.testcontainers.Testcontainers;
+import org.testcontainers.utility.MountableFile;
+
+import java.net.InterfaceAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.testcontainers.shaded.org.awaitility.Awaitility.await;
+
+/**
+ * Integration tests for {@link RabbitmqAccountCreatedEventSender}.
+ */
+@SpringBootTest(classes = GeorchestraGatewayApplication.class)
+@TestPropertySource(properties = { //
+        "georchestra.datadir=${user.dir}/src/test/resources/data_directory" })
```

Comment:
It does not sound to me as very portable, it doestn't resolve on my dev environment.